site/e2e/api.ts

```diff
@@ -166,7 +169,8 @@ export async function verifyConfigFlagEntries(
 	);
 
 	// Verify array of options with green marks.
-	Object.entries(opt.value)
+	// biome-ignore lint/suspicious/noExplicitAny: opt.value is any
+	Object.entries(opt.value as any)
```

Comment:
This whole expression is a bit janky. I'll see if I can't clean up some if it during the two work days we have during Christmas break